doc: added asyncapi student ambassador md file (#1333)
asyncapi/community: STUDENT_AMBASSADOR.md

This document covers the organizational and operational aspects of the AsyncAPI Student Ambassador Program.

> ### What is the AsyncAPI Student Ambassador Program?

Reviewer: would be good to have a short sentence that explain the difference between AsyncAPI Ambassador and AsyncAPI Student Ambassador as I'm not 100% sure what it is

Reviewer 2: +1

> After seeing the impact of the AsyncAPI Ambassador Program we thought to extend this to the Student commnunity. This aims to bring AsyncAPI closer to both current users and potential new project users in the student community. It also aims to provide users and community members with the necessary tools and resources to:
> 
> - Promote user cooperation and disseminate the necessary tools to any specific user.
> - Engage with the project in some way, either as a contributor, blogger, speaker, etc.
> - Speak at community events or write technical content (i.e., blog posts) that solves community needs in various institutions. 
> - Nurture community growth and a welcoming environment to spread knowledge about the project.

Reviewer 2: ```suggestion
In light of the impact created by the AsyncAPI Ambassador Program, we have decided to extend a similar initiative to the Student community. The initiative aims to bring AsyncAPI closer to current and potential new project users in the student community. It also aims to provide users and community members with the necessary tools and resources to:

- Promote user cooperation and disseminate the necessary tools to any specific user.
- Engage with the project in some way, either as a contributor, blogger, speaker, etc.
- Speak at community events or write technical content (i.e., blog posts) that solves community needs in various institutions. 
- Nurture community growth and create a welcoming environment to spread knowledge about the project.
```

### How does the program work?

> The student ambassador selection process will be as transparent as possible. Similar to the existing ambassador program:

Reviewer 3: We can add link to the process of selection of ambassadors programs here. Or we can move the `how to nominate yourself as a student ambassador` in the last section.

Author: Thanks for that suggestion @devilkiller-ag

> The student ambassador selection process will be as transparent as possible. Similar to the existing ambassador program:
> 
> - The ambassador selection process will be as transparent as possible.

Reviewer 2: ```suggestion
Similar to the existing ambassador program, the student ambassador selection process will be as transparent as possible:
```

> - The ambassador selection process will be as transparent as possible.

Author: Thanks @Mayaleeeee

- The model currently in place for adding members to the Amabassadors through PR will be used.
- The TSC or delegated maintainers of community repositories will evaluate applications according to compliance requirements via a traditional PR flow.
- All student ambassador information will be displayed on the AsyncAPI website in a list after the PR is merged.
- The program has the team tag `@asyncapi/student-ambassadors` to notify ambassadors on GitHub.

### Student Ambassador requirements

Participation will be determined and checked annually. These are the requirement for being a student ambassador:

- Be an active member of the community.
- Be a student or be part of any student alumni club

> - Must have merged at least three PR or have written or is to make 3 contributions in the form of articles, talks, videos, podcasts, presentations, driving initiatives, etc.

Reviewer: > at least three PR

is it really enough? making 3 good first issues is not really that time consuming and difficult. Shouldn't we expect that someone first becomes maintainer/triager or that someone completes gsoc/mentorship. This way they demonstrate they contribute long enough to know the community that they will promote - you know what I mean?

for example https://www.linkedin.com/in/hridyesh-bisht/ has great credentials, he was part of GSoD that he completed, he also writes articles https://programmerprodigy.code.blog/2024/07/24/build-docs-as-code-apis/ and afaik goes to meetups with students.

you know what I mean? like code/docs contributions through PRs  - I don't think they are good measure of someone who is a candidate to be ambassador. Shouldn't we require someone to act as ambassador before we make them one?

Author: Thank you for sharing your perspective and for pointing out valid considerations. @derberg 

I understand your concern that making a few contributions may not fully capture someone's long-term commitment or deep understanding of the community. Our aim is to lower the entry barriers while still encouraging sustained and meaningful engagement, as discussed in one of our meetings. However, we also recognize that becoming an ambassador involves more than just technical contributions—it’s about representing and actively promoting the community.

That said, your suggestion of looking for additional engagement, such as completing mentorship programs or becoming a maintainer, is something we should definitely consider as a part of our evaluation process. It’s important for ambassadors to embody our values and deeply understand our ecosystem. I don’t just think it should be mandatory.

Wdyt?

Reviewer 4: I second @derberg, summed up to "some prior experience" right ?

Reviewer: I'm ok with lowering barier, but ambassador needs to be someone who is able to say something about AsyncAPI

in case of AsyncAPI Ambassadors, we are able to evaluate by the work people do to promote the spec and tools

in case of Student Ambassador, we need people that will promote the community even more - so they need to know something about it.

completion of mentorships is one thing that could be on the list. We can have multiple items that could be used to evaluate. To see someone was helping with some community activities - maybe actively help organize conference, or run some other "jobs"

- Have a good understanding of AsyncAPI tools and specifications.
- Have an interest in mentoring others.
- Enthusiastic about teamwork and collaboration
- Passionate about people, specifically the members and how to support them.
- Accept the AsyncAPI (code of conduct)[https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md]

> Ambassadors can be invited to review content (i.e., articles, talks, videos) related to the community. These reviews are optional and will be counted as an additional contribution.

Reviewer 5: Is this also applicable to student ambassadors

Author: Is it not? @thulieblack

Reviewer 5: Maybe but it would be best to specify

Some clarifications on the type of contributions:
    
- Articles, videos, and podcasts can be published on [asyncapi.com/blog](https://www.asyncapi.com/blog) or other domains. 
- Talks and presentations can be displayed at different events or workshops. Within it, AsyncAPI should appear at the core of the presentation. The presentation would never be used as a disguised marketing tool for other products/projects.
- Special contributions should be driven to support the community or enhance its visibility (i.e., getting new sponsors or driving initiatives) 
- If we did not identify the contributions, just let us know.
         
### Student Ambassador Structure

School Leaders: These are  ambassadors leading all the developers in their schools. They coordinate the regular school activities in regard to the AsyncAPI need

Skill-Based Team Leads: These are ambassadors leading different skill-based teams, such as design team leads, content team leads, etc. The ambassadors in these teams are responsible to their team leads.

> AsycAPI Community Manager: Ultimately, ambassadors are directly under the AsyncAPI Ambassadors Program. Issues and concerns should be reported to the Community Manager.

Reviewer 5: ```suggestion
AsyncAPI Community Manager: Ultimately, ambassadors are directly under the AsyncAPI Ambassadors Program. Issues and concerns should be reported to the Community Manager.
```

Author: Seen. I will check it again

   
### Student Ambassadors duties

- Be in tune with AsyncAPI's mission and values.
- Always respect the [code of conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
- Ready to engage the members in their local community
- Be active in your role as an ambassador.

#### Other duties include:

- Oversee the activities and engagement of your local community
- Honing your data analysis skills by learning performance measurement.
- Creating activities that will grow the AsyncAPI community
- Making contributions in the form of articles, talks, videos, podcasts, presentations, driving initiatives, etc.

### Student Ambassador benefits

- Invitation to [AsyncAPI Initiative organization](https://github.com/orgs/asyncapi/people).
- Receive a special swag pack for Ambassadors (valued at $150 approx., including shipping cost).

> - Receive swag for conference or workshop participants (valued at $70 approx., including shipping cost).

Reviewer 5: Please double check this information

Author: When you say double check, do you mean for the swags or the invitation link? @thulieblack

Reviewer 6: For the swags we should double check .

Author: Alright

Reviewer: yeah, with current store we already know that this is unrealistic. Sending even small package to for example India, costs over $100

but don't worry about current store - we are closing right? and alternative will be to have a distributed net of swag providers @iambami you will need to work on it on case by case scenarion - for example: someone needs a package of stickers? based in Nigeria? - first time you will have to do some effort to find a good, reliable provided that can print and deliver locally - then document it and in future use this provider - does it make sense?

### Difference between AsyncAPI Ambassador and AsyncAPI Student Ambassador
The AsyncAPI Ambassador and AsyncAPI Student Ambassador programs are both designed to promote and support the AsyncAPI initiative but cater to different audiences and have distinct roles, responsibilities, and requirements. AsyncAPI Ambassador is open to professionals or anyone passionate about the AsyncAPI initiative, while the AsyncAPI Student Ambassador program is geared towards students and recent graduates who are interested in sharing the AsyncAPI knowledge with their institution and alumni community.

### How does the program work?
The student ambassador selection process will be as transparent as possible. Similar to the existing ambassador program:

- The model currently in place for adding members to the Ambassadors through PR will be used.
- The TSC or delegated maintainers of community repositories, will evaluate applications according to compliance requirements via a traditional PR flow.
- All student ambassador information will be displayed on the AsyncAPI website in a list after the PR is merged.
- The program has the team tag `@asyncapi/student-ambassadors` to notify ambassadors on GitHub

> ### Student Ambassador requirements

Reviewer: have you look at comment from @ALIPHATICHYD about his experience with Postman Ambassador program. I quite like the requirements:

- candidate uploads to YT a video where they explain some AsyncAPI concept
- they need to first organize some event or do presentation at their university about AsyncAPI

Since student ambassador operates at schools, and is a first point of contact, do events and explains basics - the above requirements make a lot of sense

I wonder how other companies do their programs and what requirements they have.

YT is cool thought - we could require it is any platform as long as it is short like YT Shorts or TicTok or any other form of short video

Author: Yes, I totally agree. All we have to do is give it a duration standard. It could be on YouTube or any other platform, probably just a video uploaded to Google Drive. But since we don’t use TikTok, I think YouTube or Drive will be better.

Some companies require videos to explain why they think they are fit for the position or what inspires them to apply. However, I think the approach @ALIPHATICHYD shared is more appropriate for this.

Participation will be determined and checked annually. These are the requirements for being a student ambassador:

- Be an active member of the community.
- Be a student or be part of any student alumni club
- Must have completed any of the mentorship programs, either GSOC or AsyncAPI Mentorship program 
- Have a good understanding of AsyncAPI tools and specifications.
- Have an interest in mentoring others.
- Enthusiastic about teamwork and collaboration
- Passionate about people, specifically the members, and how to support them.

> - Accept the AsyncAPI (code of conduct)[https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md]

Reviewer 5: ```suggestion
- Accept the AsyncAPI [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md)
```

Some clarifications on the type of contributions:
    
- Articles, videos, and podcasts can be published on [asyncapi.com/blog](https://www.asyncapi.com/blog) or other domains. 
- Talks and presentations can be displayed at different events or workshops. Within it, AsyncAPI should appear at the core of the presentation. The presentation would never be used as a disguised marketing tool for other products/projects.
- Special contributions should be driven to support the community or enhance its visibility (i.e., getting new sponsors or driving initiatives) 
- If we did not identify the contributions, just let us know.
         
### Student Ambassador Structure

School Leaders: These are ambassadors leading all the developers in their schools. They coordinate the regular school activities regarding the AsyncAPI need

Skill-Based Team Leads: These are ambassadors leading different skill-based teams, such as design team leads, content team leads, etc. The ambassadors in these teams are responsible for their team leads.

> AsycAPI Community Manager: Ultimately, ambassadors are directly under the AsyncAPI Ambassadors Program. Issues and concerns should be reported to the Community Manager.

Reviewer 5: ```suggestion
AsyncAPI Community Manager: Ultimately, student ambassadors are directly under the AsyncAPI Ambassadors Program. Issues and concerns should be reported to the Community Manager.
```

   
### Student Ambassadors duties

- Be in tune with AsyncAPI's mission and values.
- Always respect the [code of conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
- Ready to engage the members in their local community
- Be active in your role as an ambassador.

#### Other duties include:

- Oversee the activities and engagement of your local community
- Honing your data analysis skills by learning performance measurement.
- Creating activities that will grow the AsyncAPI community
- Making contributions in the form of articles, talks, videos, podcasts, presentations, driving initiatives, etc.

### Student Ambassador benefits

- Invitation to [AsyncAPI Initiative organization](https://github.com/orgs/asyncapi/people).
- Receive a special swag pack for Ambassadors 
- Free entry to AsyncAPI conferences.
- Community-wide recognition.
- Community badge on GitHub
- All our respects for your contribution!

### Become a Student AsyncAPI Ambassador

> The process of becoming a student ambassador is easy. Go to the **community** repository and open a PR to edit the `ambassadors.json` file. The TSC will then evaluate your candidacy!

Reviewer 5: I think we should have a separate ```.json``` file for student ambassadors so it's not confusing
